### PR TITLE
Added Repeatable and Sequential character check

### PR DESCRIPTION
**Pull request by mdeloughry**
Added

- Repeatable character checks any characters that occur 3 or more fail.
- Sequential character checks that any alphanumeric characters aren't present both forward and in reverse.

### CODE REVIEW

**Comment by mdeloughry** [comment]
Yeah very good suggestion I'll do this now along with style changes

File: src/Rules/SequentialCharacters.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param string $attribute
+     * @param mixed  $value
+     *
+     * @return bool
+     */
+    public function passes($attribute, $value)
```
Comment (dextermb) [inline src/Rules/SequentialCharacters.php]: This is good for an initial rule. I think [my suggestion](https://github.com/langleyfoxall/laravel-nist-password-rules/issues/16#issuecomment-507964062) would be more performant.

We can always improve and tweak things later on down the line. Let's get this in for now 👍

Comment (mdeloughry) [reply src/Rules/SequentialCharacters.php]: @dextermb oh sorry I hadn't seen this before commit and submitting a PR I can implement this now if we want? I'm happy to do that.

and @DivineOmega I've gone ahead and made the change you requested.

Comment (dextermb) [reply src/Rules/SequentialCharacters.php]: > @dextermb oh sorry I hadn't seen this before commit and submitting a PR I can implement this now if we want?

Either is fine 👍 I'm sure stuff will be tweaked as the package continues to be used either way.

Comment (DivineOmega) [reply src/Rules/SequentialCharacters.php]: Let's get this merged as is. We can tweak it later on if necessary.

**Pull request merged**